compile/compile.go

```diff
@@ -474,6 +475,8 @@ func (c *Compiler) compilePlan(ctx context.Context) error {
 			return err
 		}
 
+		queryPaths.Add(ast.StringTerm(c.entrypoints[i]))
```

Comment:
Instead of building this set here, call `c.compiler.GetRulesExeact(c.entrypointrefs[i])` here and check if the result is non-empty to test whether there is a ruleset defined at the entrypoint path.

One thing to consider is whether want the flag semantics to mean the entrypoint refers directly to the decision produced by a rule; or if we want to relax it a bit so they could refer to a package. If we want the latter, we could use GetRules instead.

Reply:
Neat, thanks! We use package names as entrypoints when we [compile](https://github.com/johanfylling/opa-compliance-test) the "YAML test suite" to plans for Jarl (or other IR implementations) to use for compliance tests. Would be good if we could keep it working that way :)